Jellyfin.Api/Controllers/UserController.cs

```diff
@@ -287,7 +287,8 @@ public ActionResult<AuthenticationResult> AuthenticateWithQuickConnect([FromBody
                     request.CurrentPw,
                     request.CurrentPw,
                     HttpContext.GetNormalizedRemoteIp().ToString(),
-                    false).ConfigureAwait(false);
+                    false,
+                    ignoreParentalSchedule: true).ConfigureAwait(false);
```

Comment:
Wouldn't it be better to skip `AuthenticateUser` entirely if the caller is Admin?

Reply:
IMHO it would make sense for the admin to always be able to override the password without knowing the current one.